pkg/frontend/admin_openshiftcluster_redeployvm.go

```diff
@@ -6,55 +6,20 @@ package frontend
 import (
 	"context"
 	"net/http"
-	"path/filepath"
-	"strings"
 
-	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
-
-	"github.com/Azure/ARO-RP/pkg/api"
-	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
-	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
 )
 
 func (f *frontend) postAdminOpenShiftClusterRedeployVM(w http.ResponseWriter, r *http.Request) {
-	ctx := r.Context()
-	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
-	r.URL.Path = filepath.Dir(r.URL.Path)
```

Comment:
I don't think we need to wrap these in a struct.  They're going to be used in every admin action and the context isn't limited towards just VMs.  

I'm fine with reverting these three lines and living with them in everything for now.